packages/cinterop/src/darwin/kotlin/io/realm/interop/RealmEnums.kt

```diff
@@ -74,3 +76,55 @@ actual enum class SchemaValidationMode(override val nativeValue: UInt) : NativeE
     RLM_SCHEMA_VALIDATION_SYNC(realm_wrapper.RLM_SCHEMA_VALIDATION_SYNC),
     RLM_SCHEMA_VALIDATION_REJECT_EMBEDDED_ORPHANS(realm_wrapper.RLM_SCHEMA_VALIDATION_REJECT_EMBEDDED_ORPHANS),
 }
+
+actual enum class ErrorType(override val nativeValue: UInt) : NativeEnumerated {
+    RLM_ERR_NONE(realm_wrapper.RLM_ERR_NONE),
+    RLM_ERR_UNKNOWN(realm_wrapper.RLM_ERR_UNKNOWN),
```

Comment:
Isn't this enum defined in Core? Is there a difference between Core UNKNOWN and Kotlin UNKNOWN?

I.e do we need to be able to tell the difference in an exception message?